python_modules/dagster-webserver/dagster_webserver/cli.py

```diff
@@ -183,7 +183,7 @@ def dagster_webserver(
         logger=logger,
     ) as instance:
         # Allow the instance components to change behavior in the context of a long running server process
-        instance.optimize_for_dagit(db_statement_timeout, db_pool_recycle)
+        instance.optimize_for_dagster_webserver(db_statement_timeout, db_pool_recycle)
```

Comment:
nit: i think this could be `optimize_for_webserver`

Reply:
yes